CONTRIBUTING.md

```diff
@@ -0,0 +1,59 @@
+# Contributing to ValheimPlus
+
+## How to setup your environment for V+ development
+How to setup the development enviroment to compile ValheimPlus yourself.
+
+1. Download the [BepInEx for Valheim package](https://mega.nz/file/0UAlxQwK#47InGOb8ViI6GyBDArpbhkbMTBklXdyRSmAc4-BZpJY).
+   - Unpack into your Valheim root folder and overwrite every file when asked.
+   - This package sets up your Valheim game with BepInEx configurations specifically for mod devs. Created by [MrPurple6411](https://github.com/MrPurple6411).
+1. Download the [AssemblyPublicizer package](https://mega.nz/file/oQxEjCJI#_XPXEjwLfv9zpcF2HRakYzepMwaUXflA9txxhx4tACA).
+   - This package will create new, publicized versions of the 1assembly_*.dll` files.
+   - Repo: https://github.com/MrPurple6411/Bepinex-Tools/releases/tag/1.0.0-Publicizer by [MrPurple6411](https://github.com/MrPurple6411).
+1. Drag and drop all `assembly_*.dll` files from "\Valheim\valheim_Data\Managed\" folder onto "AssemblyPublicizer.exe". This will created a new folder called "/publicized_assemblies/".
```

Comment:
```suggestion
1. Drag and drop all `assembly_*.dll` files from "\Valheim\valheim_Data\Managed\" folder onto "AssemblyPublicizer.exe". This will create a new folder called "/publicized_assemblies/".
```